src/sortStudents.ts

```diff
@@ -1,16 +1,52 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] | string {
+  function getAvarageGrade(array: number[]): number {
+    return array.reduce((a, b) => a + b, 0) / array.length;
+  }
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  const studentsClone = [...students].sort((obj1, obj2) => {
```

Comment:
obj is not good naming , here could be student1 etc or just a b :)
```suggestion
  return [...students].sort((a, b) => {
```